src/helpers/messages.js

```diff
@@ -82,10 +82,11 @@ function getUnansweredItemsWarning(scope, runner, sync) {
             )}`;
         }
     } else if (scope === 'part') {
-        if (unansweredCount === 0) {
-            itemsCountMessage = __('You answered all %s question(s)', stats.questions.toString());
-        } else {
-            itemsCountMessage = __('You have %s unanswered question(s)', unansweredCount.toString());
+        itemsCountMessage = `<b>${__('You are about to submit this test part.')}</b><br><br>`
+        if (unansweredCount > 1) {
+            itemsCountMessage += __('There are %s unanswered questions in this part of the test.', stats.answered.toString());
+        } else if (unansweredCount === 1) {
+            itemsCountMessage += __('There is %s unanswered question in this part of the test.', stats.answered.toString());
```

Comment:
A dot in the end `. `should be removed because can be added part with flagged items